config/locales/en.yml

```diff
@@ -484,6 +485,7 @@ en:
     school_and_term: Institution/When
     student_editors: Editors
     students: Editors
+    students_taught: Editors Inspected
```

Comment:
Maybe "Total Editors" instead. "Inspected" in this context is maybe particular to Indian and British English.